packages/grafana-ui/src/components/IconButton/IconButton.story.tsx

```diff
@@ -19,9 +19,35 @@ const meta: ComponentMeta<typeof IconButton> = {
       page: mdx,
     },
   },
+  args: {
+    name: 'apps',
+    size: 'xxxl',
```

Comment:
The default on the component is `'md'` so it's probably better to set it as that here as well.

Reply:
done!